### ISSUE TITLE
geocoding-zh-86.txt updates according to http://www.chahaoba.com/

### ISSUE THREAD

**Issue by Mortimer2013** [googlei18n/libphonenumber#1453]
Now numbers start with 8617726 will all be marked as "重庆市" which is too general. For example, numbers start with 8617726[7-8][0-9] are all from "新疆乌鲁木齐市".
I updated all possible numbers start with 8617626, please merge to latest data:
861772600|天津
861772601|天津
861772602|天津
861772603|天津
861772604|天津
861772605|天津
861772606|天津
861772607|天津
861772608|天津
861772609|天津
861772610|湖南省常德市
861772611|湖南省常德市
861772612|湖南省湘潭市
861772613|湖南省湘潭市
861772614|湖南省衡阳市
861772615|湖南省衡阳市
861772616|湖南省衡阳市
861772617|湖南省益阳市
861772618|湖南省益阳市
861772619|湖南省益阳市
861772620|重庆市
861772621|重庆市
861772622|重庆市
861772623|重庆市
861772624|重庆市
861772625|重庆市
861772626|重庆市
861772627|重庆市
861772628|重庆市
861772629|重庆市
861772630|四川省南充市
861772631|四川省南充市
861772632|四川省南充市
861772633|四川省南充市
861772634|四川省南充市
861772635|四川省南充市
861772636|四川省南充市
861772637|四川省南充市
861772638|四川省南充市
861772639|四川省南充市
861772640|四川省成都市
861772641|四川省成都市
861772642|四川省成都市
861772643|四川省成都市
861772644|四川省成都市
861772645|四川省成都市
861772646|四川省成都市
861772647|四川省成都市
861772648|四川省成都市
861772649|四川省成都市
861772650|四川省德阳市
861772651|四川省德阳市
861772652|四川省德阳市
861772653|四川省德阳市
861772654|四川省德阳市
861772655|四川省巴中市
861772656|四川省内江市
861772657|四川省乐山市
861772658|四川省乐山市
861772659|四川省乐山市
861772660|重庆市
861772661|重庆市
861772662|重庆市
861772663|重庆市
861772664|重庆市
861772665|重庆市
861772666|重庆市
861772667|重庆市
861772668|重庆市
861772669|重庆市
861772670|新疆乌鲁木齐市
861772671|新疆乌鲁木齐市
861772672|新疆乌鲁木齐市
861772673|新疆乌鲁木齐市
861772674|新疆乌鲁木齐市
861772675|新疆乌鲁木齐市
861772676|新疆乌鲁木齐市
861772677|新疆乌鲁木齐市
861772678|新疆乌鲁木齐市
861772679|新疆乌鲁木齐市
861772680|新疆乌鲁木齐市
861772681|新疆乌鲁木齐市
861772682|新疆乌鲁木齐市
861772683|新疆乌鲁木齐市
861772684|新疆乌鲁木齐市
861772685|新疆乌鲁木齐市
861772686|新疆乌鲁木齐市
861772687|新疆乌鲁木齐市
861772688|新疆乌鲁木齐市
861772689|新疆乌鲁木齐市
861772690|甘肃省兰州市
861772691|甘肃省兰州市
861772692|甘肃省兰州市
861772693|甘肃省兰州市
861772694|甘肃省兰州市
861772695|甘肃省兰州市
861772696|甘肃省兰州市
861772697|甘肃省兰州市
861772698|甘肃省兰州市
861772699|甘肃省兰州市

**Comment by penmetsaa** [comment]
@Mortimer2013,
Thank you for taking the time to improve libphonenumber!

We need valid evidence to merge this data. Please see our [contribution
guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md).

**Comment by Mortimer2013** [comment]
Country/region affected: CN
Example number(s) affected: 177 26XX XXXX
The phone number range(s) to which the issue applies : 177 26XX XXXX
The type of the number(s) ("fixed-line", "mobile", "short code", etc.): mobile
The cost, if applicable ("toll-free", "premium rate", "shared cost"): NA
Supporting evidence (for example, national numbering plan, announcement from mobile carrier, news article): http://www.chahaoba.com/17726

The data is still out of date in the latest version.

**Comment by penmetsaa** [comment]
@Mortimer2013 ,
Thanks for pointing us to the website. We've accepted this issue; however, we are now in a release freeze period and may not be able to release this until the second week of January. 

You have noted down entries in format "PROVINCECITY" but on the source website, it is in format "CITY, PROVINCE". We would like to know the original source of the data. This will not stop us updating the geocoding.

Please note that you have submitted this information with the knowledge that we may use, modify, reproduce, publicly display, and distribute the information in accordance with our open source policies. Please let us know if you did not intend to submit the information for such use.

**Comment by Mortimer2013** [comment]
Thanks. I transformed the data into format "CITY, PROVINCE" according to en/86.txt.

861772600|Tianjin
861772601|Tianjin
861772602|Tianjin
861772603|Tianjin
861772604|Tianjin
861772605|Tianjin
861772606|Tianjin
861772607|Tianjin
861772608|Tianjin
861772609|Tianjin
861772610|Changde, Hunan
861772611|Changde, Hunan
861772612|Xiangtan, Hunan
861772613|Xiangtan, Hunan
861772614|Hengyang, Hunan
861772615|Hengyang, Hunan
861772616|Hengyang, Hunan
861772617|Yiyang, Hunan
861772618|Yiyang, Hunan
861772619|Yiyang, Hunan
861772620|Chongqing
861772621|Chongqing
861772622|Chongqing
861772623|Chongqing
861772624|Chongqing
861772625|Chongqing
861772626|Chongqing
861772627|Chongqing
861772628|Chongqing
861772629|Chongqing
861772630|Nanchong, Sichuan
861772631|Nanchong, Sichuan
861772632|Nanchong, Sichuan
861772633|Nanchong, Sichuan
861772634|Nanchong, Sichuan
861772635|Nanchong, Sichuan
861772636|Nanchong, Sichuan
861772637|Nanchong, Sichuan
861772638|Nanchong, Sichuan
861772639|Nanchong, Sichuan
861772640|Chengdu, Sichuan
861772641|Chengdu, Sichuan
861772642|Chengdu, Sichuan
861772643|Chengdu, Sichuan
861772644|Chengdu, Sichuan
861772645|Chengdu, Sichuan
861772646|Chengdu, Sichuan
861772647|Chengdu, Sichuan
861772648|Chengdu, Sichuan
861772649|Chengdu, Sichuan
861772650|Deyang, Sichuan
861772651|Deyang, Sichuan
861772652|Deyang, Sichuan
861772653|Deyang, Sichuan
861772654|Deyang, Sichuan
861772655|Bazhong, Sichuan
861772656|Neijiang, Sichuan
861772657|Leshan, Sichuan
861772658|Leshan, Sichuan
861772659|Leshan, Sichuan
861772660|Chongqing
861772661|Chongqing
861772662|Chongqing
861772663|Chongqing
861772664|Chongqing
861772665|Chongqing
861772666|Chongqing
861772667|Chongqing
861772668|Chongqing
861772669|Chongqing
861772670|Urumchi, Xinjiang
861772671|Urumchi, Xinjiang
861772672|Urumchi, Xinjiang
861772673|Urumchi, Xinjiang
861772674|Urumchi, Xinjiang
861772675|Urumchi, Xinjiang
861772676|Urumchi, Xinjiang
861772677|Urumchi, Xinjiang
861772678|Urumchi, Xinjiang
861772679|Urumchi, Xinjiang
861772680|Urumchi, Xinjiang
861772681|Urumchi, Xinjiang
861772682|Urumchi, Xinjiang
861772683|Urumchi, Xinjiang
861772684|Urumchi, Xinjiang
861772685|Urumchi, Xinjiang
861772686|Urumchi, Xinjiang
861772687|Urumchi, Xinjiang
861772688|Urumchi, Xinjiang
861772689|Urumchi, Xinjiang
861772690|Lanzhou, Gansu
861772691|Lanzhou, Gansu
861772692|Lanzhou, Gansu
861772693|Lanzhou, Gansu
861772694|Lanzhou, Gansu
861772695|Lanzhou, Gansu
861772696|Lanzhou, Gansu
861772697|Lanzhou, Gansu
861772698|Lanzhou, Gansu
861772699|Lanzhou, Gansu

The data is public in China and can be used in the library. I also noticed that PhoneNumberMetadata.xml has already referenced www.chahaoba.com :-D.
